bin/fillTestData/Experiences.js

```diff
@@ -0,0 +1,343 @@
+/**
+ * Required dependencies
+ */
+const _ = require('lodash');
+const path = require('path');
+const mongooseService = require(path.resolve('./config/lib/mongoose'));
+const chalk = require('chalk');
+const yargs = require('yargs');
+const faker = require('faker');
+const mongoose = require('mongoose');
+const PromisePool = require('es6-promise-pool');
```

Comment:
We also have [`async`](https://www.npmjs.com/package/async) available in the codebase, which has methods like [`queue`](https://caolan.github.io/async/v3/docs.html#queue).

Not opposed using the pool helper but just pointing out existing things in case you'd wanna consider those. :-)

Simpler approach (that might be enough here?) is [Mongo's `insertMany()`](https://mongoosejs.com/docs/api.html#model_Model.insertMany) that has some built-in controls for parallelization.

Reply:
Cool, thanks for giving these hints! Using async/queue now.